CHANGELOG.md

```diff
@@ -68,6 +72,7 @@ add explicit VM resources `vcd_vapp_vm`.
   * `initscript`
   * `ovf`
   * `accept_all_eulas`
+* `vcd_independent_disk` deprecate fields `org` and `size`
```

Comment:
- `size` we deprecate because we need to switch from MB to Bytes
- But why `org` is being deprecated?

Reply:
Because we don't use or need it. Disk depends to vdc, not Org. https://code.vmware.com/apis/287/vcloud/doc/doc/operations/POST-CreateDisk.html

Reply:
org deprecation removed